lib/utils/meta_util.rb

```diff
@@ -50,13 +50,31 @@ def self.prepend_url_protocol(image_url)
           uri.to_s
         end
 
+        #
+        def self.search_for_first_image_in_content(content)
+          search_for_first_markdownified_image_url_in_content(content) ||
+          search_for_first_html_image_url_in_page_content(content)
```

Comment:
these two methods seem really similar in what they do. Could you just call `find_first_img` and pass in the context?